[MIG] component: Migration to 14.0

eantones opened the pull request:
Some migration adjustments:
- Fix Python dependencies
- Fix deprecation warnings

bealdav committed:
MIGR 14.0 component

bealdav committed:
FIX component: add py3.8 backport to make test right

bealdav commented on component/static/description/index.html:
```diff
@@ -367,7 +367,7 @@ <h1 class="title">Components</h1>
 !! changes will be overwritten.                   !!
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
-<p><a class="reference external" href="http://www.gnu.org/licenses/lgpl-3.0-standalone.html"><img alt="License: LGPL-3" src="https://img.shields.io/badge/licence-LGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/connector/tree/13.0/component"><img alt="OCA/connector" src="https://img.shields.io/badge/github-OCA%2Fconnector-lightgray.png?logo=github" /></a> <a cla ...
+<p><a class="reference external" href="http://www.gnu.org/licenses/lgpl-3.0-standalone.html"><img alt="License: LGPL-3" src="https://img.shields.io/badge/licence-LGPL--3-blue.png" /></a> <a class="reference external" href="https://github.com/OCA/connector/tree/14.0/component"><img alt="OCA/connector" src="https://img.shields.io/badge/github-OCA%2Fconnector-lightgray.png?logo=github" /></a> <a cla ...
```
This file is recomputed by after merge, useless update, I think

The same for readme.rst

bealdav replied:
Thanks for your proposal

eantones replied:
@bealdav you're right but my scripts generates always these files, it's harmless :-)

eantones committed:
[MIG] component: Migration to 14.0

Some migration adjustments:
- Fix Python dependencies
- Fix deprecation warnings

eantones committed:
Merge branch '14-component' into 14-component
